controllers/ovndbcluster_controller.go

```diff
@@ -533,9 +533,10 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	//
 	// Ensure the ovndbcluster headless service Exists
 	//
+	headlessServiceLabels := util.MergeMaps(serviceLabels, map[string]string{"type": "cluster"})
```

Comment:
type: headless

Reply:
this should probably be a const in v1betav1/ module

Reply:
Thanks Yatin and Ihar, will apply changes